x/staking/client/cli/cli_test.go

```diff
@@ -1260,6 +1263,75 @@ func (s *IntegrationTestSuite) TestNewCmdUnbond() {
 	}
 }
 
+// TestBlockResults tests that the validator updates correctly show when
+// calling the /block_results RPC endpoint.
+// ref: https://github.com/cosmos/cosmos-sdk/issues/7401.
+func (s *IntegrationTestSuite) TestBlockResults() {
```

Comment:
please mark this test as not short (It's not an unit test). This can be done in the suite level if all this tests are not short / not unit tests: using `Setup` or when initializing the test in the _normal_ `Test*(t *testing.T)` function.

To mark the test long / not unit-test you can use:
```
    if testing.Short() {
        t.Skip("skipping test in unit-tests mode.")
    }
```